apps/article/stylesheets/amp.styl

```diff
@@ -0,0 +1,55 @@
+- amp-header-height = 54px
+
+.body-amp
+  max-width 580px
+
+  .amp-header
+    height amp-header-height
+    border-bottom 1px solid gray-lighter-color
+    display flex
+    align-items center
+    a
+      margin-left 20px
+
+  .article-header
+    margin-top 50px
+
+  amp-carousel
```

Comment:
I doubt we'll be publishing other AMP pages that would conflict with this, but I'll go ahead and note that it would be ideal to target these with namespaced classes rather than tag names, e.g. `.article-amp-carousel`.